docs/rules/aura-service.md

```diff
@@ -0,0 +1,26 @@
+# Validate Aura Service
+
+This rule verifies that the Aura service is welly defined.
```

Comment:
```suggestion
This rule verifies that the Aura service is well defined.
```

Reply:
Oups sorry, I will fix the spell checks

Reply:
Hummm I am sad, because documentation around libraries / services are into the PDF but I cannot find them into the salesforce documentation.

I will check if I could have a link into the online pdf

Regards